afew/MailMover.py

```diff
@@ -72,31 +75,45 @@ def move(self, maildir, rules):
                     if self.dry_run:
                         continue
                     try:
-                        shutil.copy2(fname, self.get_new_name(fname, destination))
-                        to_delete_fnames.append(fname)
-                    except shutil.SameFileError:
-                        logging.warn("trying to move '{}' onto itself".format(fname))
-                        continue
+                        logging.info("Copy original mail: "+fname)
+                        logging.info("Path of mail: "+os.path.dirname(fname))
+                        logging.info("Copy to: "+destination)
+                        dbpath = self.db.get_path()
+                        logging.info("Path of DB: "+dbpath)
+                        relpath=os.path.relpath(os.path.dirname(destination), dbpath)
```

Comment:
typo - should be `realpath` and `os.path.realpath`.
Is realpath calculation even necessary?

Reply:
In the beginning I was trying to use a relative path as explained in the doc, but after some time stuck with that I finally test and successfully made it work with full path.
So no this is not more necessary I will remove it

Reply:
Oh wow, there's a `os.path.relpath` as well - didn't know that.
But if the code now works with both relative and absolute paths, we shouldn't do this here, right.

Reply:
Yes I discovered `relpath` aswell really useful to generate relative path.
I agree we can get rid of that now we only use absolute path.